support/templates/appsignal.js.erb

```diff
@@ -2,8 +2,12 @@
 
 import Appsignal from "@appsignal/javascript";
 
-export default new Appsignal({
+const appsignal = new Appsignal({
   key: "<%= @frontend_key %>",
   revision: "<%= @revision %>",
   uri: "<%= @uri %>"
 });
+
+export default appsignal;
+
+globalThis.appsignal = appsignal;
```

Comment:
I'm not sure if this fits for `globalThis`, but what about calling the global property `__APPSIGNAL__` as in the Node.js integration?

Reply:
To be clear, this is only for our convenience, to inspect the object and play around with it manually, from the browser's console in the test setup. I don't mind calling it `__APPSIGNAL__`, but the idea is for it to be easy to remember, so you just open the console and go `appsignal.sendError()` and it works.

We _may_ want to make the client globally accessible in the actual integration tho 🤔

Reply:
+1 to both things.

- Keeping it `.appsignal`
- Adding it to the actual integration.

I'm not sure it'll be as useful as in the Node.js integration but is good to have it for the sake of consistency.

Reply:
Created an issue for the second part: https://github.com/appsignal/appsignal-javascript/issues/598